frontend/pages/admin/index.vue

```diff
@@ -14,35 +14,36 @@
     <v-tab-item value="allowances">
       <v-card flat>
         <v-card-text>
-          <Allowance />
+          <Allowance :allowances="accounts" />
         </v-card-text>
       </v-card>
     </v-tab-item>
 
     <v-tab-item value="distributionConfigs">
       <v-card flat>
         <v-card-text>
-          <DistributionConfig />
+          <DistributionConfig :distributionConfigs="distributionConfigs" />
         </v-card-text>
       </v-card>
     </v-tab-item>
-    
   </v-tabs>
 </template>
 
 <script>
-import Allowance from "~/components/admin/allowance.vue";
-import DistributionConfig from "~/components/admin/distributionConfig.vue";
+import { mapGetters } from 'vuex'
+import Allowance from '~/components/admin/allowance.vue'
+import DistributionConfig from '~/components/admin/distributionConfig.vue'
 
 export default {
   components: {
     Allowance,
     DistributionConfig
   },
-  data() {
-    return {}
-  }
-};
+  computed: mapGetters({
```

Comment:
The page also need to tell the store to fetch the data it needs.
Something like should be added:

```suggestion
  fetch: async ({ store }) => {
    await store.dispatch('token/FETCH_ALLOWANCES')
    await store.dispatch('token/FETCH_DISTRIBUTION_CONFIGS')
  },
  computed: mapGetters({
```